bedrock/newsletter/templates/newsletter/updated.html

```diff
@@ -1,80 +1,81 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
 {% extends 'newsletter/base.html' %}
 
 {% set_lang_files "mozorg/newsletters" %}
 
 {# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
 {% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
 
-{% block body_id %}newsletter-updated{% endblock body_id %}
-
 {% block content %}
-  {% if unsubscribed_all %}  {# user requested to unsub all #}
-
-    <div id="main-feature" class="unsub">
+<main role="main">
+  <div class="mzp-l-content mzp-t-narrow">
+    {% if unsubscribed_all %}  {# user requested to unsub all #}
+    <section class="c-updated-block">
       <h1>{{ _('You’ve been unsubscribed.') }}</h1>
-      <h3>{{ _('We’re sorry to see you go.')}}</h3>
-    </div>
-
-    <div id="content" class="unsub billboard">
-      <h4>{{_('Would you mind telling us why you’re leaving?') }}</h4>
-
-      <form action="{{ url('newsletter.updated') }}" method="post">
-        <input type="hidden" name="unsub" value="2" />
-        <input type="hidden" name="token" value="{{ token }}" />
-        <table class="table">
-          {% for i, reason in reasons %}
+      <h2>{{ _('We’re sorry to see you go.')}}</h2>
+
+      <div class="c-updated-form">
+        <h4>{{_('Would you mind telling us why you’re leaving?') }}</h4>
+        <form action="{{ url('newsletter.updated') }}" method="post">
+          <input type="hidden" name="unsub" value="2" />
+          <input type="hidden" name="token" value="{{ token }}" />
+          <table>
+            {% for i, reason in reasons %}
+              <tr>
+                <td><label for="unsub{{ i }}">{{ reason }}</label></td>
+                <td><input type="checkbox" id="unsub{{ i }}" name="reason{{ i }}"></td>
+              </tr>
+            {% endfor %}
             <tr>
-            <td><input type="checkbox" id="unsub{{ i }}" name="reason{{ i }}"></td>
-            <td><label for="unsub{{ i }}">{{ reason }}</label></td>
+              <!-- <td><input type="checkbox" id="unsub99" name="reason-text-p"></td> -->
```

Comment:
@alexgibson I'm unsure what this checkbox input does or if we still need it?

Reply:
I think this is fine to leave as-is.